docs/02-hugo.Rmd

```diff
@@ -1,28 +1,89 @@
 
-What does a typical file and directory structure look like?
+
+Hugo uses a consistent file and folder structure to create your website.  The rest of this chapter will give more detail on the following files and folders:
 
 - config.toml
 - content
 - static
 - themes
 - layouts
 
+```{r lithium, fig.cap='Files and Folders created when you create a new_site() using blogdown', echo=FALSE, fig.align='center', out.width='90%'}
+knitr::include_graphics('images/folder-structure.png')
+```
+
 ## Configuration
 
-TOML syntax
+The first file that Hugo uses to generate a website is the configuration or config file. It's worth mentioning here that technically speaking, Hugo could generate your website without a config file if your files and folders are in the correct locations. Regardless, Hugo will look for a config file in the root directory of your website, so it's best to make sure that you have one. Generally speaking, this is the only file that Hugo expects to be in the root directory. 
+
+As its name suggests, the config file is where you tell Hugo exactly how to render your webpage and display your menus. It's also the location of other global parameters like links to your social networks, your site's name, and the base URL for your website. 
+
+When generating your site, Hugo will search for a file called config.toml first. If it doesn't find one, it will continue to search for either config.yaml or config.json. Since most themes automatically generate a config.toml file, we will discuss that here. 
+
+### TOML Syntax
+
+If you're not familiar with TOML Syntax, don't worry, we'll give a brief overview here (and you can find more information in the [documentation](https://github.com/toml-lang/toml)). TOML is made up of key and value pairs separated by an equals sign. 
+
+```{r eval=FALSE}
+key = value
+```
+
+When you want to edit a configuration in the TOML file, simply change the value. Values that are character strings should be in quotes, whereas boolean values should be lowercase and bare. 
+
+For example, if you want to give your website the title "My Awesome Site", but disable the feature to pluralize list titles (i.e., turn 'post' to 'posts') you may have the following entries in your config.toml file.
+
+```js
+title = "My Awesome Site"
+
+pluralizeListTitles = false
+```
+Most of your website's global variables are entered in the config.toml file in exactly this manner. 
+
+Further into your config file you may notice some values in brackets like this:
+
+```js
+[social]
+    github          = "https://github.com/rstudio/blogdown"
+    twitter         = "https://twitter.com/rstudio"
```

Comment:
Considering that we will print hardcopies of this book, we'd better make lines as short as possible in code blocks, otherwise they may exceed the page margin. In this case, you can remove the excessive white spaces before `=`.